src/utils/helpers/formatNumbersToString.ts

```diff
@@ -1,5 +1,5 @@
 export function formatEmissions(emissions: number): string {
-    return emissions.toFixed(2) + " grams of CO2";
+    return emissions.toFixed(2) + " g of CO2";
```

Comment:
You might want to leave it as grams for now, I think we had a discussion that grams is clearer (open to challenges)